product_docs/docs/pem/8/installing/linux_x86_64/index.mdx

```diff
@@ -13,16 +13,12 @@ redirects:
 navigation:
   - pem_rhel_9
   - pem_rhel_8
-  - pem_rhel_7
   - pem_other_linux_9
   - pem_other_linux_8
-  - pem_centos_7
   - pem_sles_15
-  - pem_sles_12
```

Comment:
SLES is still in support until November isn't it?

Reply:
Yes, it is. We will reinstate SLES 12 changes back in sometime